loadable_apps/loadable_sample/wifiapp/wifiapp.c

```diff
@@ -83,35 +83,36 @@ int wifiapp_main(int argc, char **argv)
 #ifdef CONFIG_EXAMPLES_LOADABLE_MANUAL_TEST
 	while (is_testing) {
 		display_test_scenario();
-		ch = getchar();
-		switch (ch) {
+		while ((ch = getchar()) != EOF) {
```

Comment:
I think additional while is not necessary, just break above while is enough if getchar returns EOF

Reply:
Yes, but svace tool makes error about below.
```
Integer value received from 'fgetc' can't be safely converted to a character at wifiapp.c:89 without a test for EOF.
```
So I modified like this.